README.md

```diff
@@ -4,6 +4,39 @@ This repository benchmarks performance of various Ethereum development
 frameworks by simulating a call to Convex's `systemShutdown` method. This method
 uses about 16M gas and performs a number of token transfers
 
+## Benchmarks
+
+Benchmarks were run at mainnet block 14,445,961. Blocknative only simulates against the latest block (support for simulation at historical blocks is planned), so the Blocknative simulation was run first and other benchmarks were run against the same block.
+
+Notes on benchmarks:
+- "Remote RPC" benchmarks used Alchemy as the RPC provider.
+- "Local RPC" benchmarks used a local Erigon node.
+- "Cached" benchmarks use RPC responses that the framework caches locally instead of making HTTP requests. We assume that since Blocknative and Tenderly are infrastructure providers, they aggressively cache and load hot data in-memory to improve performance, which is why their results are also in the "Cached" column.
+- Benchmarks were performed on macOS 11.6.2 with a 2.3 GHz 8-Core Intel Core i9 and 32 GB 2667 MHz DDR4.
+
+Notes on gas usage:
+- Ganache, Hardhat, and Tenderly all agree on gas usage after refunds and are therefore likely the truth value.
+- Blocknative's gas usage does not account for refunds as this is not yet supported by their platform.
+- Foundry and Dapptools exclude the 21,064 intrinsic gas from the reported gas used.
```

Comment:
.. "and have small amount of additional overhead from their contract based setups and solidity generated checks."